### PR TITLE
Adding details for notifications, create jobs and list printer shares

### PR DESCRIPTION
**Pull request by nilakhan**
Adding details for notifications, create jobs and list printer shares

### CODE REVIEW

File: api-reference/beta/api/print-list-shares.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ GET /print/shares
 ## Optional query parameters
 This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
 
-To see a list of each printer share's capabilities, include the optional `$select=capabilities` query parameter.
+> **Note:** Using $top=n query parameter will return **upto** `n` shares. Caller needs to use skip token to enumerate over the entire list.
```
Comment (braedenp-msft) [inline api-reference/beta/api/print-list-shares.md:38]: nit: "up to" (2 words)

File: api-reference/beta/api/print-list-shares.md [hunk 2]
```diff
@@ -47,8 +47,18 @@ Some operators are not supported: `$count`, `$orderby`, `$search`.
 
 ## Request body
 Do not supply a request body for this method.
+
 ## Response
 If successful, this method returns a `200 OK` response code and a collection of [printerShare](../resources/printershare.md) objects in the response body.
+
+>**Note**: The response will not contain the **defaults** and **capabilities** properties.
```
Comment (braedenp-msft) [inline api-reference/beta/api/print-list-shares.md:54]: nit: "or" instead of "and"

File: api-reference/beta/api/printer-post-jobs.md [hunk 3]
```diff
@@ -15,6 +15,8 @@ Namespace: microsoft.graph
 
 Create a new [printJob](../resources/printJob.md) for a [printer](../resources/printer.md). 
 
+> **Note:** A user can submit upto ~10000 print jobs in 10 days.
```
Comment (braedenp-msft) [inline api-reference/beta/api/printer-post-jobs.md:18]: nit: "up to"

File: api-reference/beta/api/printershare-post-jobs.md [hunk 4]
```diff
@@ -15,6 +15,8 @@ Namespace: microsoft.graph
 
 Create a new [printJob](../resources/printJob.md) for a [printerShare](../resources/printerShare.md). 
 
+> **Note:** A user can submit upto ~10000 print jobs in 10 days.
```
Comment (braedenp-msft) [inline api-reference/beta/api/printershare-post-jobs.md:18]: nit: "up to"

File: api-reference/v1.0/api/print-list-shares.md [hunk 5]
```diff
@@ -36,10 +36,10 @@ GET /print/shares
 ## Optional query parameters
 This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
 
-To see a list of each printer share's capabilities, include the optional `$select=capabilities` query parameter.
+> **Note:** Using $top=n query parameter will return **upto** `n` shares. Caller needs to use skip token to enumerate over the entire list.
```
Comment (braedenp-msft) [inline api-reference/v1.0/api/print-list-shares.md:39]: nit: "up to"

File: api-reference/v1.0/api/print-list-shares.md [hunk 6]
```diff
@@ -53,7 +53,13 @@ Do not supply a request body for this method.
 
 If successful, this method returns a `200 OK` response code and a collection of [printerShare](../resources/printershare.md) objects in the response body.
 
->**Note**: The response will not contain the **defaults** and **capabilities** properties. You can get these properties via a [Get printerShare](printershare-get.md) request.
+>**Note**: The response will not contain the **defaults** and **capabilities** properties.
```
Comment (braedenp-msft) [inline api-reference/v1.0/api/print-list-shares.md:56]: nit: "or" instead of "and"

File: api-reference/v1.0/api/printer-post-jobs.md [hunk 7]
```diff
@@ -14,6 +14,8 @@ Create a new [printJob](../resources/printJob.md) for a [printer](../resources/p
 
 Also creates a new [printDocument](../resources/printDocument.md) associated with the printJob.
 
+> **Note:** A user can submit upto ~10000 print jobs in 10 days.
```
Comment (braedenp-msft) [inline api-reference/v1.0/api/printer-post-jobs.md:17]: nit: "up to"

File: api-reference/v1.0/api/printershare-post-jobs.md [hunk 8]
```diff
@@ -14,6 +14,8 @@ Create a new [printJob](../resources/printJob.md) for a [printerShare](../resour
 
 Also creates a new [printDocument](../resources/printDocument.md) associated with the printJob.
 
+> **Note:** A user can submit upto ~10000 print jobs in 10 days.
```
Comment (braedenp-msft) [inline api-reference/v1.0/api/printershare-post-jobs.md:17]: nit: "up to"

**Review by braedenp-msft — APPROVED** [review]

**Review by Lauragra — APPROVED** [review]

**Pull request merged**